multimodal/tests/unittests/test_mmocr_text_detection.py

```diff
@@ -42,10 +42,10 @@ def test_mmocr_text_detection_inference(checkpoint_name):
     MMOCR_res = ocr.readtext(mmocr_image_name, output=None)
 
     # compare the outputs of original model's output and our model
-    assert len(pred) == len(MMOCR_res[0]["boundary_result"])  # num of text lines
+    assert len(pred[0]) == len(MMOCR_res[0]["boundary_result"])  # num of text lines
```

Comment:
If the first dimension of pred result is not used, we may consider remove it in extract_from_output

Reply:
Done.